src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -120,6 +121,7 @@ public string InstalledVersions
         private async Task UpdateInstalledVersionsAsync(CancellationToken cancellationToken)
         {
             var hash = new HashSet<NuGetVersion>();
+            var vulnerabilitiesHash = new HashSet<NuGetVersion>();
```

Comment:
Or simply `vulnerabilities`. The Hash part of the HashSet is an implementation detail of the specific kind of Set being used and implies it's the vulnerabilities Hashed. What we're really interested in is a unique, unordered list of vulnerabilities, i.e. a Set.

```suggestion
            var vulnerabilitiesSet = new HashSet<NuGetVersion>();
```